syntax = "proto3";

message TreeNode {
  int32 level = 1;
  int32 childSize = 2;
  double minx = 3;
  double maxx = 4;
  double miny = 5;
  double maxy = 6;
  double minz = 7;
  double maxz = 8;
  int32 slice_id = 9;
}

message Tree {
  repeated TreeNode treeNodes = 1;
}